src/main/features/response/views/check-and-send.njk

```diff
@@ -26,7 +26,22 @@
         }
       )
     }}
-    {{ row(label = 'Contact number (optional)', value = draft.defendantDetails.mobilePhone.number, changeLink = ResponsePaths.defendantMobilePage.evaluateUri({ externalId: claim.externalId }), bold = true) }}
+
+    {% if claim.claimData.defendant.phone %}
+      {{ row(
```

Comment:
As this is on the same page as the confirm your name and address details section it should be included in there. So there should be no change link and it should only have a bottom border and remove the border from the other section above.